cli/hiera.go

```diff
@@ -111,6 +111,8 @@ func NewCommand() *cobra.Command {
 		`a key:value or key=value where value is literal expressed using Puppet DSL`)
 	flags.StringArrayVar(&cmdOpts.FactPaths, `facts`, nil,
 		`like --vars but will also make variables available under the "facts" (for compatibility with Puppet's ruby version of Hiera)`)
+	flags.BoolVar(&cmdOpts.MultiLookup, `multi-lookup`, false,
+		`lookup all of the keys and output the results as a map`)
```

Comment:
Suggest that this is named `--all` instead of `--multi-lookup`. This because the standard lookup is also "multi", but it stops at the first found key (IIRC), whereas this new option looks up "all" - just like the help text says.
(Also `lookup --multi-lookup ...` looks a bit heavy).